producer/data_repository.go

```diff
@@ -87,7 +129,10 @@ func AddEntrySmPolicyTable(imsi string, dnn string, snssai *protos.NSSAI) error
 		Sd:  snssai.Sd,
 		Sst: int32(sval),
 	}
-	smPolicyData := MongoDBLibrary.RestfulAPIGetOne(collName, filter)
+	smPolicyData, errGetOne := commonDBClient.RestfulAPIGetOne(collName, filter)
+	if errGetOne != nil {
+		logger.DataRepoLog.Errorln(errGetOne)
+	}
```

Comment:
should we return ? or we are allowing it to go further and log the message?

Reply:
I think `return` is not necessary here. We can just log and go further. In the rest of the code, if clauses checks the availability of smPolicyData and progress it accordingly.